dashboard/test/ui/features/project_gallery.feature

```diff
@@ -0,0 +1,45 @@
+# TODO (Erin B.) remove ?showFeatured=1 param from project gallery url during flag removal.
+
+@dashboard_db_access
+Feature: Public Project Gallery
+
+  Background:
```

Comment:
nit: background should be indented the same as Feature and Scenario for consistency